r2dbc-mysql/src/main/java/JasyncStatement.kt

```diff
@@ -71,6 +72,11 @@ internal class JasyncStatement(private val clientSupplier: Supplier<JasyncConnec
         return this
     }
 
+    fun releaseAfterwards(): Statement {
```

Comment:
how can this be used? it's not part of the interface. by casting to JasyncStatement?

Reply:
Yes, that would be the idea, if the programmer was really keen on making sure that the prepared statement is closed they could cast it and request for it to be closed, for example in Java 14:

```
	private void releaseAfterUse(Statement st) {
		if (st instanceof JasyncStatement jst) jst.releasePreparedStatementAfterUse();
	}
```